### DISCUSSION TITLE
Arrow Buttons Rework

### DISCUSSION THREAD

**Discussion by EmmetSchuler** [lichess-org/mobile#391] [Ideas]
I really appreciate the design of the Mobile V2 app, but something that slightly bothers me is the placement and size of the Arrow Buttons to move around in a chess game.
For my hands, even on a 6.4 inch screen, I feel like I keep missing the arrow buttons when navigating my games. I'm pretty sure this is due to the small size of the hit area for the arrow buttons.
I would appreciate a larger hit area for the navigation keys, to avoid missing the buttons, even when it looks as if you are clicking on them.

https://github.com/lichess-org/mobile/assets/108140722/c640e86f-ea2f-4d1a-9326-0a3e5873e4a0

In the video you can see the small size of the clickable area for the arrow buttons.

There also seems to be inconsistencies in the spacing between the arrows, as seen in the game analysis screen and the game itself, the latter shown below.

![Screenshot_20231214-201935.png](https://github.com/lichess-org/mobile/assets/108140722/57b6cd67-6ed0-4797-927e-c1f22bb4b913)

I appreciate all the hard work everyone is putting in to make this app happen!

**Comment by veloce** [comment]
Should be good now with the last release so I'm closing this.
